h/static/scripts/annotator/anchoring/pdf.coffee

```diff
@@ -249,6 +251,10 @@ exports.describe = (root, range, options = {}) ->
   startTextLayer = getNodeTextLayer(range.start)
   endTextLayer = getNodeTextLayer(range.end)
 
+  if startTextLayer is null
```

Comment:
When combined with your change earlier in this file, I guess this means that the root cause of this bug is that `range.start` is `null`. But why is `range.start` null?